tsl/test/expected/data_node.out

```diff
@@ -172,6 +172,21 @@ SELECT * FROM add_data_node('data_node_3', host => 'localhost', database => :'DN
  data_node_3 | localhost | 55432 | db_data_node_3 | t            | t                | t
 (1 row)
 
+SET ROLE :ROLE_1;
+-- Create a distributed hypertable where no nodes can be selected
+-- because there are no data nodes with the right permissions.
+CREATE TABLE disttable(time timestamptz, device int, temp float);
+\set ON_ERROR_STOP 0
+\set VERBOSITY default
+SELECT * FROM create_distributed_hypertable('disttable', 'time', 'device');
+NOTICE:  3 of 3 data nodes not used by this hypertable due to lack of permissions
+HINT:  Grant USAGE on data nodes to attach them to a hypertable.
+ERROR:  no data nodes can be assigned to the hypertable
+DETAIL:  Data nodes existed, but none had USAGE privilege.
```

Comment:
I am wondering if "Data nodes exist, but none have USAGE privilege" is better than the above?

Reply:
Yeah, that is better. Fixed.